src/CommitmentsV2/SFA.DAS.CommitmentsV2.Api.Types/Responses/ApprenticeshipDetails.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using SFA.DAS.CommitmentsV2.Types;
+
+namespace SFA.DAS.CommitmentsV2.Api.Types.Responses
+{
+    public class ApprenticeshipDetails
```

Comment:
A pattern we've adopted recently is to nest types like this into the "parent" response type, to avoid attempted reuse and prevent name/space clashes. So you could shift this entire class into the GetApprenticeshipsResponse class.

Reply:
ok cool i'll change it